### PR TITLE
Updated static and const

### PR DESCRIPTION
**Pull request by mani-chand**
Updated the content for space issue .
#1464 issue.

### CODE REVIEW

File: src/user-defined-types/const.md
Comment (randomPoison) [inline src/user-defined-types/const.md:3]: One minor request: Could you change the wording here to not refer to constants as "variables"? That was a minor tweak I've been meaning to make, and it seems reasonable to do it here since you're reworking the pages already.

```suggestion
Constants are evaluated at compile time and their values are inlined
```

File: src/user-defined-types/static.md
Comment (randomPoison) [inline src/user-defined-types/static.md:8]: Can you remove this heading? This slide is about statics, and I think this little introductory header is unnecessary clutter.

File: src/user-defined-types/static.md
Comment (randomPoison) [inline src/user-defined-types/static.md:32]:
```suggestion
- `static` is similar to mutable global variables in C++.
```

File: src/SUMMARY.md
Comment (randomPoison) [inline src/SUMMARY.md:67]:
```suggestion
  - [Statics](user-defined-types/static.md)
  - [Constants](user-defined-types/const.md)
```

Per my suggestion, could you remove the properties slide and then remove the extra level of nesting here? The title of the first slide can also change to just "Statics" since it's not introducing both concepts anymore.

File: src/user-defined-types/properties.md
Comment (randomPoison) [inline src/user-defined-types/properties.md:1]: I don't think we need this properties slide. This info was originally in the speaker notes and it felt pretty superfluous to me. I think we can remove this content, and move the "More to Explore" section to the speaker notes on the statics slide.

**Review by randomPoison — CHANGES_REQUESTED** [review]
This mostly looks good, I just have a few minor requests for tweaks.

File: src/SUMMARY.md
Comment (djmitche) [inline src/SUMMARY.md:65]: Don't forget to add a redirect in `book.toml` for the old path.

**Review by djmitche — APPROVED** [review]
LGTM otherwise, pending @randomPoison's approval.

**Review by randomPoison — APPROVED** [review]

**Comment by djmitche** [comment]
Oh, I forgot about book.toml. I'll add that now.

**Pull request merged**
